selftests/functional/runner_vmimage.py

```diff
@@ -0,0 +1,211 @@
+import os
```

Comment:
Since downloading the vmimages is time-consuming, we shouldn't have run them in our regular self tests. I would propose to move them to https://github.com/avocado-framework/avocado/blob/master/selftests/pre_release/tests/ (or maybe since we have more vmimage type tests to create new folder for them) and update the [vmimagetest-config](https://github.com/avocado-framework/avocado/blob/master/selftests/check.py#L747) accordingly.